packages/expo-cli/src/easJson.ts

```diff
@@ -24,6 +24,7 @@ export interface AndroidManagedBuildProfile {
   workflow: Workflow.Managed;
   credentialsSource: CredentialsSource;
   buildType?: 'apk' | 'app-bundle';
+  releaseChannel?: undefined;
```

Comment:
why undefined?

Reply:
@wkozyra95 so that TS doesn't complain. When generic project has it and managed doesn't, accessing this property on the union type says that can't access the property.